virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc

```diff
@@ -10,6 +10,8 @@ A boot source contains a bootable operating system (OS) and all of the configura
 
 You use a boot source to create virtual machine templates with specific configurations. These templates can be used to create any number of available virtual machines.
 
+Quick Start tours are available in the {product-title} web console to assist you in creating a custom boot source, uploading a boot source, and other tasks.
```

Comment:
I think it would be helpful to add a sentence: "Select **Quick Starts** from the **Help** menu to view the Quick Start tours."

Reply:
LGTM

Reply:
Done.